README.md

```diff
@@ -73,55 +85,145 @@ This will clone `10.15.6` to a new VM and, if there are differences from the bas
 
 > Check out the [examples directory](./examples) to see how port-forwarding and other options are used
 
-## Configuration
+## Builders 
+
+### veertu-anka-vm-create
+
+#### Required Configuration
+
+* `installer_app` (String)
+
+The path to a macOS installer. This process takes about 20 minutes. The resulting VM template name will be `{{vm_name}}-{{macOSVersion}}`. macOSVersion is pulled from the installer app.
+
+* `type` (String)
+
+Must be `veertu-anka-vm-create`.
+
+* `vm_name` (String)
+
+The name for the VM that is created. One is generated if not provided (`anka-packer-{10RandomCharacters}`).
 
-* `type` (required)
+#### Optional Configuration
 
-Must be `veertu-anka`.
+* `anka_password` (String)
 
-* `installer_app` (optional)
+Sets the password for the vm. Can also be set with `ANKA_DEFAULT_PASSWD` env var. Defaults to `admin`.
 
-The path to a macOS installer. This must be provided if `source_vm_name` isn't provided. This process takes about 20 minutes. The resulting VM template name will be `anka-packer-base-{macOSVersion}`.
+* `anka_user` (String)
 
-* `disk_size` (optional)
+Sets the username for the vm. Can also be set with `ANKA_DEFAULT_USER` env var. Defaults to `anka`.
+
+* `boot_delay` (String)
+
+The time to wait before running packer provisioner commands, defaults to `10s`.
+
+* `cpu_count` (String)
+
+The number of CPU cores, defaults to `2`.
+
+* `disk_size` (String)
 
 The size in "[0-9]+G" format, defaults to `25G`.
 
 > We will automatically resize the internal disk for you by executing: `diskutil apfs resizeContainer disk1 0`
 
-* `ram_size` (optional)
+* `hw_uuid` (String)
+
+The Hardware UUID you wish to set (usually generated with `uuidgen`).
+
+* `port_forwarding_rules` (Struct)
+
+> If port forwarding rules are already set and you want to not have them fail the packer build, use `packer build --force`
+
+```json
+  "builders": [{
+    "type": "veertu-anka-vm-clone",
+    "source_vm_name": "anka-packer-base-10.15.7",
+    "port_forwarding_rules": [
+      {
+        "port_forwarding_guest_port": 80,
+        "port_forwarding_host_port": 12345,
+        "port_forwarding_rule_name": "website"
+      },
+      {
+        "port_forwarding_guest_port": 8080
+      }
+    ],
+    "vm_name": "macos-from-packer-base_10.15.7"
+  }]
+```
+
+* `ram_size` (String)
 
 The size in "[0-9]+G" format, defaults to `2G`.
 
-* `cpu_count` (optional)
+* `stop_vm` (Boolean)
 
-The number of CPU cores, defaults to `2`.
+Whether or not to stop the vm after it has been created, defaults to false.
```

Comment:
This setting might be a good place to handle the logic associated with the limited capabilities of the develop license. Rather than having separate stop logic, we either a) force the correct value here or b) raise an error if you try to specify suspend but are using an unsupported license. My vote is for (a).